tests/integration/golang/aim/project/get_project_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"testing"
+	"fmt"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetProjectTestSuite struct {
+	suite.Suite
+	client          *helpers.HttpClient
+	projectFixtures *fixtures.ProjectFixtures
+	project         *fiber.Map
+}
+
+func TestGetProjectTestSuite(t *testing.T) {
+	suite.Run(t, new(GetProjectTestSuite))
+}
+
+func (s *GetProjectTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	projectFixtures, err := fixtures.NewProjectFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.projectFixtures = projectFixtures
+
+	project := projectFixtures.GetProject(context.Background())
+	s.project = project
+}
+
+func (s *GetProjectTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.projectFixtures.UnloadFixtures())
+	}()
+	var resp fiber.Map
```

Comment:
I think we should convert to a struct in the `response` package

Reply:
can we create type `response.GetProjects` which can hold this response, instead of fiber.Map?